src/operations_problems_templates.jl

```diff
@@ -147,10 +157,20 @@ uc_problem = UnitCommitmentProblem(system)
 - `devices::Dict{Symbol, DeviceModel}` : override default `DeviceModel` settings
 - `branches::Dict{Symbol, DeviceModel}` : override default `DeviceModel` settings
 - `services::Dict{Symbol, ServiceModel}` : override default `ServiceModel` settings
+- Key word arguments supported by `OperationsProblem`
 """
 
 function UnitCommitmentProblem(system::PSY.System; kwargs...)
-    template = template_unit_commitment(; kwargs...)
+    kwargs = Dict(kwargs)
+    template_kwargs = Dict()
+    for kw in (:network, :devices, :branches, :services)
```

Comment:
same here. it is better to make a list by saying `not in OPERATIONS_ACCEPTED_KWARGS`